tests/integration/targets/ec2_asg_lifecycle_hook/inventory

```diff
@@ -0,0 +1,6 @@
+[tests]
+create_update_delete
```

Comment:
Since we're only running one test, do we need the more complex framework for running in parallel?